<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewContact</class>
 <widget class="QWidget" name="NewContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Whisper - Registration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/new/prefix1/Resource/logo.png</normaloff>:/new/prefix1/Resource/logo.png</iconset>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>82</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(32, 101, 144);
border-radius: 8px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>452</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 68px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/new/prefix1/Resource/down_reg.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>142</y>
     <width>431</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255,140);
border-radius: 8px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>92</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR DESTINE</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
color: rgb(151, 184, 201);
background-color: rgb(255, 255, 255,0);
</string>
   </property>
   <property name="text">
    <string>REGISTRATION</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>22</y>
     <width>651</width>
     <height>561</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/new/prefix1/Resource/jins-reg.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="accept_button">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>468</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(255,255, 200,80);
border-width: 1px;
     border-style: solid;
     border-color:  rgb(255, 255, 255, 200);</string>
   </property>
   <property name="text">
    <string>ACCEPT</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>238</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sex:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>284</y>
     <width>35</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>330</y>
     <width>34</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>City:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>379</y>
     <width>77</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>185</y>
     <width>88</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enter_user_name">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>185</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(22, 0, 200,10);
border-width: 1px;
     border-style: solid;
     border-color:  rgb(255, 255, 255, 200);</string>
   </property>
   <property name="maxLength">
    <number>14</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="sex_person">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>238</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(161, 188, 199);</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Unknown</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Man</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Woman</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="age">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>284</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(161, 188, 199);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enter_city">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>330</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(22, 0, 200,10);
border-width: 1px;
     border-style: solid;
     border-color:  rgb(255, 255, 255, 200);</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="enter_password">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>379</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(22, 0, 200,10);
border-width: 1px;
     border-style: solid;
     border-color:  rgb(255, 255, 255, 200);</string>
   </property>
   <property name="maxLength">
    <number>14</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Error_label">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>211</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(214, 71, 0);</string>
   </property>
   <property name="text">
    <string>Error.. Name already exists.</string>
   </property>
  </widget>
  <widget class="QLabel" name="Error_label_2">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>410</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(214, 71, 0);</string>
   </property>
   <property name="text">
    <string>Error.. Enter username and password.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>462</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
border-width: 1px;
border-style: solid;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/Resource/back.png</normaloff>:/new/prefix1/Resource/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/new/prefix1/Resource/123.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="minimazregBut">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/Resource/minbt.png</normaloff>:/new/prefix1/Resource/minbt.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="closeregBut">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>3</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/Resource/clbt.png</normaloff>:/new/prefix1/Resource/clbt.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>label_6</zorder>
  <zorder>label_2</zorder>
  <zorder>label_10</zorder>
  <zorder>accept_button</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>sex_person</zorder>
  <zorder>age</zorder>
  <zorder>enter_user_name</zorder>
  <zorder>enter_city</zorder>
  <zorder>enter_password</zorder>
  <zorder>Error_label</zorder>
  <zorder>Error_label_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_9</zorder>
  <zorder>minimazregBut</zorder>
  <zorder>closeregBut</zorder>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
